kubernetes_deploy/development/jobs.yml

```diff
@@ -0,0 +1,20 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  annotations:
+    kubernetes.io/change-cause: "<to be filled in deploy_to_kubernetes script>"
+  name: laa-legal-adviser-api-collect-static
+spec:
+  template:
+    metadata:
+      name: laa-legal-adviser-collect-static
+    spec:
+      restartPolicy: Never
+      containers:
+      - name: app
+        image: "<to be set by deploy_to_kubernetes>"
+        imagePullPolicy: Never
+        command: ["python", "manage.py", "collectstatic", "--noinput"]
```

Comment:
What do you think about using `docker/collectstatic.sh`?

Reply:
I think it's an unnecessary extra layer tbh

Reply:
I found entry scripts to be an eloquent way to represent concepts with meaningful names and for reuse across Docker, local machine and CI. For example, it was easy to use `bin/migrate` even in CI because it was so obvious what it did.

That said, I'm fine not using it if the domain means it's easy to reproduce from top of our heads, but in that case, can we remove the file, please?

Reply:
Can we remove it already? I figured it was in use by template deploy

Reply:
I don't think it is. template-deploy should copy files directly from the container.
Ah, in that case, this pull request would break it :(

Reply:
Oh you're right, it was added for running locally on k8s, so is now redundant and I've now removed it.
If we want to have standardised entry points I wouldn't be opposed to that, but that feels like a separate piece of work now.